news/2018-03-06-new-featured-artist-trial-and-error.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "New Featured Artist: Trial & Error"
+date: 2018-03-06 11:00:00 +0000
+---
+
+The search for musical talent never ends, and we're proud to announce the next addition to osu!'s Featured Artist lineup - **Trial & Error**!
+
+[![](https://assets.ppy.sh/artists/20/header.jpg)](https://osu.ppy.sh/beatmaps/artists/20)
+
+The mind behind countless Japanese and anime-themed tracks, **Trial & Error**'s work has been featured in popular projects like the cult classic visual novel *Stay! Stay! Democratic People's Republic of Korea* and *Panzermadels: Tank Dating Simulator*. 
```

Comment:
visual novels: *Stay! Stay! Democratic People's Republic of Korea* and *Panzermadels: Tank Dating Simulator*